wgsl/index.bs

```diff
@@ -15556,3 +15573,61 @@ Use token definitions from the main part of the specification.
 <pre class=include>
 path: wgsl.recursive.bs.include
 </pre>
+
+# Appendix A: The `text/wgsl` media type # {#text-wgsl-media-type}
```

Comment:
This is copied from the already-reviewed media-type-registration.txt file, but with updated markup.